src/main/resources/io/jenkins/plugins/agent_build_history/agentBuildHistory.css

```diff
@@ -90,3 +90,27 @@
   gap: 18px;
   padding: 0 15px;
 }
+
+.abh-pagination-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+  margin-top: 20px;
```

Comment:
the `margin-top` is not needed
instead add some minimal margin to the bottom so the inputs are a but separated from the table below